update/imsi.py

```diff
@@ -138,14 +134,7 @@ def cleanup_value(val):
 # This matches a line containing a MCC/MNC, e.g.
 # | 232 || 02 || || A1 Telekom Austria || Reserved || ||
 _mnc_line_re = re.compile(
-    r'^\|\s*(?P<mcc>[0-9]+)' +
-    r'\s*\\\\\s*(?P<mnc>[0-9]+)' +
-    r'(\s*\\\\\s*(?P<brand>[^\\]*)' +
-    r'(\s*\\\\\s*(?P<operator>[^\\]*)' +
-    r'(\s*\\\\\s*(?P<status>[^\\]*)' +
-    r'(\s*\\\\\s*(?P<bands>[^\\]*)' +
-    r'(\s*\\\\\s*(?P<notes>[^\\]*)' +
-    r')?)?)?)?)?')
+    "^\|\s*(?P<mcc>[0-9]+)\s*\\\\\s*(?P<mnc>[0-9]+)"+)'}(\s*\\\\\s*(?P<brand>[^\\]*)"*)'}(\s*\\\\\s*(?P<operator>[^\\]*)"*)'}(\s*\\\\\s*(?P<status>[^\\]*)"*)'}(\s*\\\\\s*(?P<bands>[^\\]*)"*)'}(\s*\\\\\s*(?P<notes>[^\\]*)"*)'})?)?)?)?)?")
```

Comment:
I don't think this makes the code more readable. The original regex already has some readability issues but at least the splitting over multiple lines tries to make it a little better readable.

Also flake8 gives some errors.